src/pages/EventEditPage/event_edit.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import {
+  getEventById,
+  setEventDetails,
+  timestampToDate,
+  dateToTimestamp,
+} from '../../services/events';
+import EventEditForm from '../../components/EventEditForm';
+
+class EventEditPage extends React.Component {
+  constructor(props) {
+    super(props);
+    const {
+      match: {
+        params: { eventId },
+      },
+    } = props;
+    this.state = { eventId, initialValues: {}, formLoading: true };
+  }
+
+  componentDidMount() {
+    const { eventId } = this.state;
+
+    getEventById(eventId).then(event => {
+      const initialValues = {
+        ...event,
+        startDate: timestampToDate(event.startDate),

```

Comment:
leaky abstraction - should put timestamp conversion within service since timestamp implementation is specifc to firebase. 
@thaigillespie